examples/basic-integration-jetpack-compose/app/src/main/java/com/deuna/compose_demo/view_models/HomeViewModel.kt

```diff
@@ -200,6 +196,42 @@ class HomeViewModel(private val deunaSDK: DeunaSDK) : ViewModel() {
     }
 
 
+    fun clickToPay(
```

Comment:
@darwinmorocho-deuna dejar comment de que es click to pay y porque tener que tener esta funcion aparte.

@darwinmorocho-deuna por otro lado, puedes actualizar el readme de iOS y Android?

Reply:
@esteban-deuna  defini esta función para evitar confusiones entre el ejemplo de implementación del vault y click to pay